### PR TITLE
UX: Various improvements to welcome topic CTA

### PR DESCRIPTION
**Pull request by nbianca**
The welcome topic CTA has been moved to an user tip that is displayed when an admin navigates to that topic. The topic has also been rewritten to be friendlier and will be displayed in bootstrap mode even if it was not edited by an admin.

image

### CODE REVIEW

**Review by Sammypit — COMMENTED** [review]
Many kinds of getting back to me thank you so much I'd like to keep in contact please

**Comment by jjaffeux** [comment]
Im not sure the title of the commit reflects the content of the PR which seems to primarily delete code?

**Comment by nbianca** [comment]
Yes, the welcome topic has been revamped completely and quite a few changes happened, but the PR description already enumerates exactly what those improvements are.

**Comment by nbianca** [comment]
Yes, the welcome topic has been revamped completely and quite a few changes happened, but the PR description already enumerates exactly what those improvements are.

File: app/assets/javascripts/discourse/app/lib/user-tips.js [hunk 1]
```diff
@@ -25,6 +26,7 @@ export function showUserTip(options) {
     trigger: "manual",
     theme: "user-tips",
     zIndex: "",
+    delay: isTesting() ? 0 : 100,
```
Comment (pmusaraj) [inline app/assets/javascripts/discourse/app/lib/user-tips.js:29]: Just curious, any reason for the delay?

File: config/locales/client.en.yml [hunk 2]
```diff
@@ -1930,6 +1930,10 @@ en:
         title: "Keep reading!"
         content: "Here are some topics we think you might like to read next."
 
+      welcome_topic:
+        title: "Welcome topic"
+        content: "As an admin, you can edit this welcome topic and make it more welcoming to your community members!"
```
Comment (pmusaraj) [inline config/locales/client.en.yml:1935]: There's a bit of repetition here (also with the title of topic). Maybe a small copyedit can help: 

```suggestion
        title: "Edit the welcome topic"
        content: "Help new members feel at home by customizing this topic to suit your community's needs."
```

**Review by pmusaraj — APPROVED** [review]
Looks good to me, just a few minor notes.

File: app/assets/javascripts/discourse/app/lib/user-tips.js (hunk 1 above)
Comment (nbianca) [reply app/assets/javascripts/discourse/app/lib/user-tips.js:29]: No delay can cause the user tip to jump a bit while the page is rendered.

Comment (jjaffeux) [reply app/assets/javascripts/discourse/app/lib/user-tips.js:29]: Did you try wrapping this in a `schedule("afterRender", ...` block?

DOM operations should almost always happen inside this kind of block.

Comment (nbianca) [reply app/assets/javascripts/discourse/app/lib/user-tips.js:29]: Yes, I tried that. It was slightly better, but it still did not fix the issue.

Comment (nbianca) [reply app/assets/javascripts/discourse/app/lib/user-tips.js:29]: I added that now too because I find it to be a good suggestion nonetheless.

**Pull request merged**